Content.Client/DeltaV/SpaceFerret/CanBackflipSystem.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Content.Shared.DeltaV.SpaceFerret;
+using Robust.Client.Animations;
+using Robust.Client.Audio;
+using Robust.Client.GameObjects;
+using Robust.Shared.Animations;
+using Robust.Shared.Audio;
+using Robust.Shared.Player;
+
+namespace Content.Client.DeltaV.SpaceFerret;
+
+public sealed class CanBackflipSystem : EntitySystem
+{
+    [Dependency] private readonly AnimationPlayerSystem _animation = default!;
+    [Dependency] private readonly AudioSystem _audio = default!;
+
+    public const string BackflipKey = "backflip";
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeNetworkEvent<DoABackFlipEvent>(OnBackflipEvent);
+    }
+
+    public void OnBackflipEvent(DoABackFlipEvent args)
+    {
+        if (!TryGetEntity(args.Actioner, out var uid))
+        {
+            return;
+        }
+
+        _animation.Play(uid.Value, new Animation
```

Comment:
no validation of any kind here either...

Reply:
it used to let malf clients play ***any*** sound which is Not Good

now it cant do that, also can probably get mob from message session so player cant even spoof the entity of an actual ferret